diff --git a/dist/_node-chunks/chunk-HT5O7ZWA.js b/dist/_node-chunks/chunk-HT5O7ZWA.js
index a21020ab3c6b308e091f94b3e194ca2c96adf2cc..2e8eda105a01c9d0a64f5beffe677a28d4b09608 100644
--- a/dist/_node-chunks/chunk-HT5O7ZWA.js
+++ b/dist/_node-chunks/chunk-HT5O7ZWA.js
@@ -1266,7 +1266,7 @@ var getValue = (obj, key) => {
 // src/csf-tools/enrichCsf.ts
 import { generate as generate4, types as t5 } from "storybook/internal/babel";
 var enrichCsfStory = (csf, csfSource, key, options) => {
-  let storyExport = csfSource.getStoryExport(key), isCsfFactory = t5.isCallExpression(storyExport) && t5.isMemberExpression(storyExport.callee) && t5.isIdentifier(storyExport.callee.object) && storyExport.callee.object.name === "meta", source = !options?.disableSource && extractSource(storyExport), description = !options?.disableDescription && extractDescription(csfSource._storyStatements[key]), parameters = [], baseStoryObject = isCsfFactory ? t5.memberExpression(t5.identifier(key), t5.identifier("input")) : t5.identifier(key), originalParameters = t5.memberExpression(baseStoryObject, t5.identifier("parameters"));
+  let storyExport = csfSource.getStoryExport(key), isCsfFactory = t5.isCallExpression(storyExport) && t5.isMemberExpression(storyExport.callee) && (t5.isIdentifier(storyExport.callee.object) && storyExport.callee.object.name === "meta" || t5.isIdentifier(storyExport.callee.property) && storyExport.callee.property.name === "extend"), source = !options?.disableSource && extractSource(storyExport), description = !options?.disableDescription && extractDescription(csfSource._storyStatements[key]), parameters = [], baseStoryObject = isCsfFactory ? t5.memberExpression(t5.identifier(key), t5.identifier("input")) : t5.identifier(key), originalParameters = t5.memberExpression(baseStoryObject, t5.identifier("parameters"));
   parameters.push(t5.spreadElement(originalParameters));
   let optionalDocs = t5.optionalMemberExpression(
     originalParameters,
